data/human/ships.txt

```diff
@@ -2420,15 +2424,17 @@ ship "Mule"
 		"X3200 Ion Steering"
 		"Hyperdrive"
 		
-	engine -12 123
-	engine 12 123
-	gun -9 -119 "Sidewinder Missile Launcher"
-	gun 9 -119 "Sidewinder Missile Launcher"
-	turret 22 -65 "Heavy Anti-Missile Turret"
-	turret -37 -21 "Heavy Laser Turret"
-	turret 39 35 "Heavy Laser Turret"
-	turret -9 59 "Heavy Anti-Missile Turret"
-	bay "Fighter" -16 -9
+	engine -7.5 129
+	engine 11 128
+	turret 19.5 -40
+	turret -22.5 -54
+	turret 20 32
+	turret -20.5 103
+	gun -33.5 56.5
+		angle 305
+	gun -38.5 62
+		angle 305
```

Comment:
Bruh moment.

Reply:
If it needs to be changed I don't mind. I'm just waiting out how things turn out before I commit any huge changes; that was something I already had set up to be in line with the first iteration